src/libcollections/vec.rs

```diff
@@ -862,6 +862,9 @@ impl<T> Vec<T> {
     /// ```
     #[unstable(feature = "map_in_place",
                reason = "API may change to provide stronger guarantees")]
+    #[deprecated(since = "1.3.0",
+                 reason = "unclear that the API is strong enough and has \
+                           yet to prove itself")]
```

Comment:
the wording here with 'has yet' would imply that this could change in the future, I would say "did not prove itself"